examples/resources/genesyscloud_script/apis.md

```diff
@@ -0,0 +1,5 @@
+* [POST /api/v2/scripts/{scriptId}/export](https://developer.genesys.cloud/devapps/api-explorer#post-api-v2-scripts--scriptId--export)
+* [GET /api/v2/scripts](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-scripts)
```

Comment:
A private API was used to create the script
I'm not sure if I should include something like `POST https://apps.{domains}/uploads/v2/scripter`. I would just put `/uploads/v2/scripter` but I noticed it's `apps`, and not `api`

Reply:
@dginty4 @carnellj